.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Azshara.lua

```diff
@@ -1567,6 +1772,12 @@ root(ROOTS.Zones, m(KALIMDOR, {
 					["timeline"] = { ADDED_4_0_3 },
 					["races"] = HORDE_ONLY,
 					["groups"] = {
+						objective(2, {	-- 0/4 Blackmaw Warrior slain
```

Comment:
This appears to be missing objective(1), might be able to review a video or playthrough the quest to see if a particular interaction with an object causes that objective to trigger (such as  aportal)

Reply:
well first you use item then talk to npc and get teleported and then talk to npc there to trigger a fight